ntbs-service/DataMigration/NotificationImportService.cs

```diff
@@ -183,6 +183,28 @@ public class NotificationImportService : INotificationImportService
                 return importResult;
             }
 
+            var hasAnyRecordAlreadyBeenImported = false;
+            foreach (var notification in notifications)
+            {
+                // Check that record hasn't already been imported
+                var foundLtbr = await _notificationRepository.NotificationWithLegacyIdExistsAsync(notification.LTBRID);
+                var foundEts = await _notificationRepository.NotificationWithLegacyIdExistsAsync(notification.ETSID);
+                if (foundEts || foundLtbr)
+                {
+                    hasAnyRecordAlreadyBeenImported = true;
+                    var errorId = foundLtbr ? "LTBRId = " + notification.LTBRID : "ETSId = " + notification.ETSID;
```

Comment:
It would be better to use string interpolation for these values, e.g. `$"LTBRId = {notification.LTBRID}"`